packages/opentelemetry-baggage-span-processor/README.md

```diff
@@ -0,0 +1,47 @@
+# @uphold/opentelemetry-baggage-span-processor
+
+Package that sets all baggage entries as span attributes.
+
+## Installation
+
+```sh
+npm install @uphold/opentelemetry-baggage-span-processor
+```
+
+## Why?
+
+Setting all baggage entries as span attributes using OpenTelemetry ensures end-to-end traceability, simplifies analysis, and enhances observability in distributed systems. This approach provides consistent contextual information across spans, facilitating correlation, context propagation, and streamlined troubleshooting. It also enables flexibility in using analysis tools, improving operational efficiency and accelerating issue resolution.
```

Comment:
```suggestion
Finding spans or traces that match contextual data stored in baggage is not an out-of-the-box feature in OpenTelemetry. This packages sets all baggage entries as span attributes so that you can search any span based on data stored in the baggage, like a `user.id` or `request.id`.
```